package.json

```diff
@@ -12,6 +12,7 @@
     "grunt-contrib-connect": "^1.0.2",
     "grunt-contrib-copy": "~1.0.0",
     "grunt-contrib-sass": "~1.0.0",
+    "grunt-contrib-uglify": "^3.3.0",
```

Comment:
Maybe use a Tilda to match most other deps? Not really sure on this though. We should come up with a better policy for versions. @agjohnson @ericholscher How does RTD handle this?

Reply:
I think npm switched from using caret instead of tilde by default a while ago if you add a dependency using `--save-dev`.
Basically `^3.3.0` will upgrade to any `3.x.y` but not to `4.0.0`. `~3.3.0` will upgrade to any `3.3.y`, but not to `3.4.y`.
Since we're talking about semver, caret should be safe enough, but I'm fine with tilde if the extra safety is desired.

Reply:
I am fine with whatever, it just would be nice to keep consistency.

Reply:
I've changed it to tilde for consistency.